shared/tinyusb/mp_usbd.h

```diff
@@ -29,8 +29,9 @@
 
 #include "py/obj.h"
 
-// Call instead of tud_task()
-void mp_usbd_task(void);
+// Port can call this function from an IRQ context in order to pend a call
+// to the TinyUSBD task handler.
+void mp_usbd_schedule(void);
```

Comment:
This function is not defined anywhere...?

Reply:
oops, sorry, this is left over from an early version of the patch. will remove.